.circleci/config.yml

```diff
@@ -338,6 +338,7 @@ jobs:
           name: Install NPM dependencies
           command: |
             npm --version
+            npm set progress false
```

Comment:
can this be controlled by ENV ?

Reply:
Well... I think this line might be actually redundant (https://docs.npmjs.com/misc/config#progress).